articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+
+
+    (source: [siloRepository.sol](https://www.contractreader.io/contract/0xd998C35B7900b344bbBe6555cc11576942Cf309d))
+
+* setPriceProvidersRepository - The chosen oracle solution can also affect users' deposits. The DAO must ensure that to select the best oracle options for each silo.
+
+    
+
+<p id="gdcalert8" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image8.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert9">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image8.png "image_tooltip")
+
+
+
+    (source: [siloRepository.sol](https://www.contractreader.io/contract/0xd998C35B7900b344bbBe6555cc11576942Cf309d))
+
+* GuardedLaunch.sol - a contract which is part of Silo’s permission system - enables the SiloDAO (as Timelock Controller) to implement security and risk-averse functions. For instance, it allows the contract manager (onlyManager) to pause specific silos in case of an exploit. If that happens, users deposited in the paused silos are at risk of liquidation, if their collateral value decreases below the liquidation threshold, after a new start (remove pause).
+
+
+
+<p id="gdcalert9" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image9.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert10">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image9.png "image_tooltip")
+
+
+
+## (source: [SiloRepository.sol (GuardedLaunch.sol)](https://www.contractreader.io/contract/0xd998C35B7900b344bbBe6555cc11576942Cf309d))
+
+
+## Governance Risk
+
+As mentioned above, all changes that affect the Silo protocol and its parameters are implemented via an on-chain governance vote. These changes can include:
+
+
+
+* Deployment of new markets (silos),
+* Setting/adjusting silo parameters (LTV, LT, interest model, price feed),
+* Minting/redeeming XAI
+* Turning on/off fees
+* Deploying new bridge assets,
+* and even increasing the supply of SILO tokens
+
+In summary, governance has extensive and very far-reaching powers over Silo. Hence, the question arises “how likely is it that a malicious party can obtain a majority voting influence to instigate harmful changes to the protocol” (e.g. mint infinite XAI)?
+
+For this to happen, the party would need to accrue a substantial amount of SILO tokens (or get voting power delegated). Presuming that the team and SILO investors only vote in the best interest of the DAO, the answer is: The possibility is very low. The chart below shows the voting power of all delegators. It’s important to note that to participate in on-chain voting, one has to delegate SILO to one's wallet or another delegate.
```

Comment:
`possibility is very low.` however, your following assessment of token distribution suggests founding team has significant control over the protocol, so requires trust that team wont rug. Are team doxxed?